dask_sql/physical/rel/logical/distinct.py

```diff
@@ -0,0 +1,28 @@
+import logging
+from typing import TYPE_CHECKING
+
+from dask_sql.datacontainer import DataContainer
+from dask_sql.physical.rel.base import BaseRelPlugin
+
+if TYPE_CHECKING:
+    import dask_sql
+    from dask_planner.rust import LogicalPlan
+
+logger = logging.getLogger(__name__)
+
+
+class DaskDistinctPlugin(BaseRelPlugin):
+    """
+    Eliminates all duplicate entries from the specified DataFrame column
+    """
+
+    class_name = "Distinct"
+
+    def convert(self, rel: "LogicalPlan", context: "dask_sql.Context") -> DataContainer:
+        (dc,) = self.assert_inputs(rel, 1, context)
+        df = dc.df
+        cc = dc.column_container
+
+        df = df.drop_duplicates()
```

Comment:
The `Distinct` plan was previously represented as an `Aggregate` with grouping expressions and no aggregate expressions. I don't know how this was being mapped to Dask but we would probably want to continue mapping in the same way. I am assuming that the failing tests are because of the change here but I could be wrong.

Reply:
ahhh ok that is good to know. I think I could just map it back to the old way then. Thanks for the info.